index.html

```diff
@@ -0,0 +1,239 @@
+        <div>
+          <div>
+          <h2>Tonic</h2>
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+      <li class="item multi-post-stories">
+        <img src="img/work4.svg" alt="">
+        <div>
+          <div>
+          <h2>Multi-Post Stories</h2>    
+          <ul class="canopy">
+            <li><h3>CANOPY</h3></li>
+            <li><img src="img/circle.svg" alt=""></li>
+            <li><p>Back End Dev</p></li>
+            <li><img src="img/circle.svg" alt=""></li>
+            <li><p>2015</p></li>
+          </ul>
+          </div>
+          <p>
+            A daily selection of privately
+            personalized reads; no accounts
+            or sign-ups required.
+          </p>
+          <ul class="languages">
+            <li><a href="">html</a></li>
+            <li><a href="">css</a></li>
+            <li><a href="">javascript</a></li>
+            <li><a href="">Ruby on rails</a></li>
+          </ul>
+          <div class="rev">
+            <button><a href="" class="btn">See Project</a></button>
+          </div>
+        </div>
+      </li>
+    </ul>
+    </section>
+  <section class="about">
+    <div class="abt_head">
+    <h2>About <br>
+        Myself</h2>
+    <p>Hello I’m a software developer!
+        I can help you build a product ,
+        feature or website Look through
+        some of my work and experience!
+        If you like what you see and have
+        a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <h3>
+        LET’S CONNECT
+      </h3>  
+      <ul class="social-media">
+        <li><a href="twiter"><img src="img/twiter.svg" alt=""></a></li>
+        <li><a href="linkedin"><img src="img/Linkedin.svg" alt=""></a></li>
+        <li><a href="medium"><img src="img/medium.svg" alt=""></a></li>
+        <li><a href="github"><img src="img/github.svg" alt=""></a></li>
+        <li><a href="chill"><img src="img/chill.svg" alt=""></a></li>
+      </ul> 
+      <button  class="btn"><a href="">Get my resume</a></button>
+    </div>
+    <div class="links2">
+      <div class="itm1">
+        <div class="lan">
+          <h2>Languages</h2>
+          <img src="imgoverlay/Union.svg" alt="">
```

Comment:
- [ ] **[Optional]** It would be better to reduce the size of this **icon** and Kindly also try to improve spacing on your page to match that of Figma design :+1: :+1: 

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/205422768-50c88462-bca0-4342-bd03-64f29990a584.png) | ![image](https://user-images.githubusercontent.com/102757126/205422780-3fd6f453-dfe7-46fb-ab82-4790e30dadc1.png)